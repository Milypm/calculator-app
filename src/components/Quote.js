import React from 'react';
import Navbar from './Navbar';
import '../style/style.css';

const Quote = () => (
  <div className="first-div">
    <Navbar />
    <h2 className="quote-h2">
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding. â€“William Paul Thurston
    </h2>
  </div>
);

export default Quote;
