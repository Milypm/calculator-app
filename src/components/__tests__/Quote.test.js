import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../Quote';

describe('Quote component', () => {
  test('renders Quote component', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Quote />
      </BrowserRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
