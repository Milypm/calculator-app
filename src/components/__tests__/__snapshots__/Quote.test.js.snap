// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders Quote component 1`] = `
<div
  className="first-div"
>
  <div
    className="header-navbar"
  >
    <div
      className="header-icon"
    >
      <h1>
        Math Magicians
      </h1>
      <i
        className="fas fa-hat-wizard"
      />
    </div>
    <ul
      className="nav-ul"
    >
      <li
        className="li-separator"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="li-separator li-calc"
      >
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote-of-the-day"
          onClick={[Function]}
        >
          Quote of the Day
        </a>
      </li>
    </ul>
  </div>
  <h2
    className="quote-h2"
  >
    "But in my opinion, all things in nature occur mathematically."â€“ Rene Descartes
  </h2>
</div>
`;
